src/components/ViewDeck/ViewDeck.js

```diff
@@ -97,228 +80,205 @@ class ViewDeck extends Component {
       Green: 0,
       Gray: 0,
     };
-    //.reduce
+    //TODO: .reduce instead
 
     cards.forEach((card) => {
-      if (card.colors.length === 0 && card.cmc > 0) {
-        console.log("colorless card with cmc of", card.cmc);
-        devotion.Gray += card.cmc;
+      if (card.jsonData.colors.length === 0 && card.jsonData.cmc > 0) {
+        //console.log("colorless card with cmc of", card.jsonData.cmc);
+        devotion.Gray += card.jsonData.cmc;
       }
 
-      for (let index = 0; index < card.mana_cost.length; index++) {
-        const element = card.mana_cost[index];
+      for (let index = 0; index < card.jsonData.mana_cost.length; index++) {
+        const element = card.jsonData.mana_cost[index];
         if (element.includes("{") || element.includes("}")) {
           //console.log('this includes a { or }');
         } else {
-          console.log("this is", element);
+          //console.log("this is", element);
           if (element === "W") {
             //console.log('white');
-            devotion.White += 1;
+            devotion.White += 1 * card.quantity;
           }
           if (element === "U") {
             //console.log('blue');
-            devotion.Blue += 1;
+            devotion.Blue += 1 * card.quantity;
           }
           if (element === "B") {
             //console.log('black');
-            devotion.Black += 1;
+            devotion.Black += 1 * card.quantity;
           }
           if (element === "R") {
             //console.log('red');
-            devotion.Red += 1;
+            devotion.Red += 1 * card.quantity;
           }
           if (element === "G") {
             //console.log('green');
-            devotion.Green += 1;
+            devotion.Green += 1 * card.quantity;
           }
         }
       }
     });
-    console.log("NEW devotion", devotion);
-
+    //console.log("NEW devotion", devotion);
     return devotion;
   };
 
-  cmc = (cards) => {
-    //console.log("🚀 ~ file: ViewDeck.js ~ line 184 ~ ViewDeck ~ cards", cards)
+  const cmcCalc = (cards) => {
     var convertedManaCosts = [];
 
     cards.forEach((card) => {
-      //console.log(card.cmc);
-      if (card.type_line.includes("land")) {
+      if (card.jsonData.type_line.includes("land")) {
         //console.log('this is a land and shouldnt be counted');
       } else {
-        convertedManaCosts.push(card.cmc);
-        //console.log(convertedManaCosts);
+        convertedManaCosts.push(card.jsonData.cmc);
       }
     });
 
+    //NOT SURE WHAT THIS WAS MEANT TO DO ORIGIONALLY
```

Comment:
I feel like there is some overlap with cmc and devotion. Maybe that is because I don't know MTG very well. Delete lines 135-138. They do nothing and you even state it in this last comment.

Reply:
Going to put this in useEffect like we talked about.  I really want to make a bar graph happen with CMC.  This is lower on my list of priorities at the moment but something I still want to do.